contrib/bundle/crio-registries.conf

```diff
@@ -0,0 +1 @@
+unqualified-search-registries = ["docker.io", "quay.io"]
```

Comment:
A drive-by comment on a PR which, I think, doesn’t change anything in this area:

Shipping two registries, both of which allow registration by ~untrusted users, in the default configuration, is a security risk: https://www.redhat.com/en/blog/be-careful-when-pulling-images-short-name

Reply:
Thanks! We added this for convenience, but can reconsider once I'm done with https://github.com/cri-o/cri-o/issues/7380